BSB_lan_defs.h

```diff
@@ -9014,7 +9019,9 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 // Diagnose Erzeuger - Solar
 {0x053D09AB,  CAT_DIAG_ERZEUGER,    VT_ONOFF,         8499,  STR8499,  sizeof(ENUM_ONOFF),   ENUM_ONOFF,   FL_RONLY,     DEV_ALL}, // Kollektorpumpe 1 (Aus) - Broetje NovoCondens WOB20-25
 {0x053D09AB,  CAT_DIAG_ERZEUGER,    VT_ONOFF,         8499,  STR8499_2,sizeof(ENUM_ONOFF),   ENUM_ONOFF,   FL_RONLY,     DEV_123_ALL}, // Kollektorpumpe 1  - Broetje EcoTherm WMS 24
+//Looks like ENUM with functions list
```

Comment:
Can you query/set this parameter on your heater?

Reply:
RGT1->HEIZ QUR 8502 ENUM_CAT_2c_TEXT - STR8502_TEXT:
DC 86 00 0B 06 3D 05 0A 8B 44 5F
HEIZ->RGT1 ANS 8502 ENUM_CAT_2c_TEXT - STR8502_TEXT: 01 00
DC 80 06 0D 07 05 3D 0A 8B 01 00 64 B3
Baxi show on display "Keine Funktion"
Unfortunately, I can't figure out how to turn on the heat accumulator or swimpool so that the value of this parameter changes

Reply:
Ah, ok, 01 00 is "disabled" (Keine Funktion), so yes, unless you have the additional equipment, you can't set these parameters, but you're right that it's most likely VT_ENUM...